api/src/main/java/org/pmiops/workbench/dataset/DataSetServiceImpl.java

```diff
@@ -789,10 +790,12 @@ private Optional<String> buildConceptIdSqlInClause(
           dbConceptSetConceptIds.stream()
               .collect(Collectors.partitioningBy(DbConceptSetConceptId::isStandard));
       String standardConceptIds =
-          partitionSourceAndStandard.get(true).stream()
-              .map(c -> c.getConceptId().toString())
-              .sorted()
-              .collect(Collectors.joining(", "));
+          splitWithLineBreaks(
```

Comment:
alt idea: could implement this as a Collector and apply at the end of the stream, to avoid the repetition of joining and later splitting

Reply:
Can you provide more detail on this idea? (the code was written by copilot...)
I was thinking this would be a large change to other SQL code, so I made that static Util so that it can be used by other `IN` in the future

Reply:
This will only fix the IN clause for a concept set, but will not change any of the other IN clauses that can be generated by the cohort builder. My suggestion would be to split the line breaks of any IN clauses in the final query that is generated.

Reply:
@freemabd that is just my idea based on my limited knowledge... And it turns out not working even for concept set. I might need dataapps help.

Reply:
I would suggest changing the sql here as it's the final generated sql statemet: https://github.com/all-of-us/workbench/blob/3af08340f49b924788e2f8a4cf5d47c06d14eef4/api/src/main/java/org/pmiops/workbench/dataset/DataSetServiceImpl.java#L1492-L1492

Reply:
This is the sql produced in the code sample that is copied for RStudio 
![Screen Shot 2024-03-14 at 9 23 30 AM](https://github.com/all-of-us/workbench/assets/3904919/6ab73bf9-61f1-4ff2-a664-0d57bb74bf5c)